content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/at_confirmation.rst

```diff
@@ -0,0 +1,86 @@
+===========================
+At confirmation reservation
+===========================
+
+.. _inventory/reservation_methods/at-confirmation:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+The *At Confirmation* reservation method reserves products **only** when a sales order (SO) is
+confirmed, **and** if enough stock of the products included in the |SO| is already available.
+
+Configuration
+=============
+
+To set the reservation method to *At Confirmation*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`. Then, select the desired :guilabel:`Operation Type` to
+configure, or create a new one by clicking :guilabel:`New`.
+
+In the :guilabel:`General` tab on the operation type form, locate the :guilabel:`Reservation Method`
+field, and select :guilabel:`At Confirmation`.
+
+.. image:: at_confirmation/at-confirmation-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+Workflow
+========
+
+To see the *At Confirmation* reservation method in action, create a new |SO| by navigating to
+:menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field. Then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order,
+which, in this case, is `100 Units`.
```

Comment:
- icon name (see screenshot, revealed in dev tools)
- icon may or may not be green for the user, would remove this status indicator (what you did with the **Note** is ok too 👍 )
- remove reference to the screenshot; the writing should be able to stand on it's own *unless* we're writing in 
this way with an **example** admonition block
- add gui for **Reserved** since it is a line item label on the tooltip
![Screenshot 2024-03-14 at 1 23 43 PM](https://github.com/odoo/documentation/assets/5496809/a8b9ba16-3fdf-4eb8-b991-07a9e929a5b1)


```suggestion
Click the :guilabel:`📈 (area graph)` icon on the product line to reveal the product's
:guilabel:`Availability` tooltip, which reveals the :guilabel:`Reserved` number of units for this order.
```

Reply:
Suggestion implemented ✅